crates/uv-fs/src/lib.rs

```diff
@@ -50,6 +50,17 @@ pub async fn read_to_string_transcode(path: impl AsRef<Path>) -> std::io::Result
 /// On Windows, this uses the `junction` crate to create a junction point.
 #[cfg(windows)]
 pub fn replace_symlink(src: impl AsRef<Path>, dst: impl AsRef<Path>) -> std::io::Result<()> {
+    // If the source is a file, we can't create a junction
+    if src.as_ref().is_file() {
```

Comment:
I don't check `is_dir()` because I don't want to enforce that it exists in case that's too constraining, happy to change if people feel strongly.